### PR TITLE
Improvements to Handbook

### PR DESCRIPTION
**Pull request by RudraSen2**
Bringing Some Improvements to Handbook...

### CODE REVIEW

File: docs/community/instances.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ You can use [DNS Lookup](https://www.ultratools.com/tools/dnsLookupResult), [Geo
 Next to its native webclient interface, Jitsi Meet is also available as
 
 * 📲 an open-source **Jitsi desktop app**, see [Jitsi Meet Electron for Desktop](https://github.com/jitsi/jitsi-meet-electron/) (Windows, Mac, and Linux clients available)
-* 📲 **mobile apps** for [Google Play](https://play.google.com/store/apps/details?id=org.jitsi.meet), [Apple Store](https://itunes.apple.com/us/app/jitsi-meet/id1165103905?mt=8), and [F-Droid](https://f-droid.org/en/packages/org.jitsi.meet/).
+* 📲 **mobile apps** for <i class="fa-brands fa-google-play"></i> [Google Play](https://play.google.com/store/apps/details?id=org.jitsi.meet), <i class="fa-brands fa-app-store-ios"></i> [Apple App Store](https://itunes.apple.com/us/app/jitsi-meet/id1165103905?mt=8), and <i class="fa-brands fa-android"></i> [F-Droid](https://f-droid.org/en/packages/org.jitsi.meet/).
```
Comment (saghul) [inline docs/community/instances.md:35]: I can't find anything about Docusaurus supporting fontawesome out of the box.

File: docs/dev-guide/mobile-jitsi-meet.md [hunk 2]
```diff
@@ -106,15 +106,17 @@ Make sure you adjust the fully qualified package name.
...
 code is being interpreted by Chrome's V8 engine, instead of JSCore which React
 Native uses. It's important to keep this in mind due to potential differences in
 supported JavaScript features.
+:::
 
 ## Enabling extra features
 
-* [Dropbox integration](mobile-dropbox.md)
-* [Google sign-in integration (for YouTube live streaming)](mobile-google-auth.md)
+- [DropBox Integration](mobile-dropbox.md)
```
Comment (saghul) [inline docs/dev-guide/mobile-jitsi-meet.md:118]: The proper capitalization is "Dropbox"

File: docs/user-guide/browsers.md [hunk 3]
```diff
@@ -11,16 +11,16 @@ Browsers evolve very rapidly, so this table may become inaccurate as new bugs on
...
 | Browser | Support | Notes |
 |---|---|---|
-| Chromium based | ✅ | We support the current stable - 10 versions |
-| Firefox | ✅ | We support current ESR - 1 versions |
-| Safari | ✅ | We support current - 1 versions |
-| Internet Explorer | ❌ | Are you serious? |
+| Chromium-Based (<i class="fa-brands fa-chrome"></i> Chrome, <i class="fa-brands fa-edge"></i> Edge, etc. ) | ✅ | We Support the Current Stable and Previous **10** Versions |
+| <i class="fa-brands fa-firefox-browser"></i> Firefox | ✅ | We Support the Current Extended Support Release and Previous 1 Version(s) |
+| <i class="fa-brands fa-safari"></i> Safari | ✅ | We Support Current and Previous 1 Version(s) |
+| <i class="fa-brands fa-internet-explorer"></i> Internet Explorer | ❌ | Are you serious? [Internet Explorer Support Ended - See this](https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-end-of-support) |
```
Comment (saghul) [inline docs/user-guide/browsers.md:17]: Drop the extra explanation on IE please, it never supported WebRTC, there was a plugin and we dropped support for that a long time ago.

File: docusaurus.config.js [hunk 4]
```diff
@@ -11,6 +11,10 @@ module.exports = {
   favicon: "img/favicon.png",
   onBrokenLinks: "throw",
   onBrokenMarkdownLinks: "warn",
+  scripts: [
+    // Please Update this with your FontAwesome Kit URL!
+    "https://kit.fontawesome.com/307bcbc229.js",
```
Comment (saghul) [inline docusaurus.config.js:16]: Ah, this is where the FontAwesome integration comes from! Nice!

File: docusaurus.config.js [hunk 5]
```diff
@@ -145,7 +152,12 @@ module.exports = {
           ],
         },
       ],
-      copyright: "Copyright © 8x8, Inc.",
+      logo: {
+        alt: "8x8 Footer Logo",
+        src: "https://web-cdn.jitsi.net/meet8x8com_6214.2958/images/8x8-copyright-icon.svg",
+        href: "/",
+      },
+      copyright: `Copyright © ${new Date().getFullYear()} 8x8, Inc.`,
```
Comment (saghul) [inline docusaurus.config.js:160]: Drop the year please.

File: docusaurus.config.js [hunk 6]
```diff
@@ -145,7 +152,12 @@ module.exports = {
           ],
         },
       ],
-      copyright: "Copyright © 8x8, Inc.",
+      logo: {
+        alt: "8x8 Footer Logo",
+        src: "https://web-cdn.jitsi.net/meet8x8com_6214.2958/images/8x8-copyright-icon.svg",
```
Comment (saghul) [inline docusaurus.config.js:157]: This URL will not be valid forever, please add the icon as a static asset here.

File: docusaurus.config.js [hunk 7]
```diff
@@ -145,7 +152,12 @@ module.exports = {
...
+        href: "/",
```
Comment (saghul) [inline docusaurus.config.js:158]: Not sure this is a good link destination...

**Review by saghul — COMMENTED** [review]
This is awesome, thank you! Left some comments, can you please take a look? I'll create that FontAwesome kit in the meantime.

**Comment by saghul** [comment]
> Will do this tomorrow surely. Today it's too late...

No rush, thank you!

File: docusaurus.config.js (hunk 7 above)
Comment (RudraSen2) [reply docusaurus.config.js:158]: Where should I put it? jitsi.org or 8x8.vc?

File: docs/community/instances.md (hunk 1 above)
Comment (RudraSen2) [reply docs/community/instances.md:35]: What should I do?

Comment (saghul) [reply docs/community/instances.md:35]: Disregard, I saw the kit stuff now.

File: docusaurus.config.js (hunk 7 above)
Comment (saghul) [reply docusaurus.config.js:158]: 8x8.com

**Comment by RudraSen2** [comment]
I have fulfilled all the requested changes. Please review again that needs to be addressed.

**Comment by saghul** [comment]
Thank you! I'll give it a quick test tonight!

**Comment by saghul** [comment]
I don't see the linked svg logo in the repo, did you forget to add it?

**Comment by RudraSen2** [comment]
> I don't see the linked svg logo in the repo, did you forget to add it?

There were just some errors in git. Now the SVG logo is added

File: docusaurus.config.js [hunk 8]
```diff
@@ -145,7 +151,12 @@ module.exports = {
           ],
         },
       ],
-      copyright: "Copyright © 8x8, Inc.",
+      logo: {
+        alt: "8x8 Footer Logo",
+        src: "/static/img/8x8-copyright-icon.svg",
```
Comment (saghul) [inline docusaurus.config.js:156]:
```suggestion
        src: "img/8x8-copyright-icon.svg",
```

**Review by saghul — APPROVED** [review]

**Pull request merged**